generators/client/templates/angular/src/main/webapp/content/scss/global.scss.ejs

```diff
@@ -31,14 +31,7 @@ h4 {
     font-weight: 300;
 }
 
-a {
```

Comment:
Instead, can we make these conditional on `clientTheme === 'none'` ?

Reply:
If there is no client theme, then the default Bootstrap styling is applied. It should again be the responsibility of this default styling to have good contrast and not of JHipster. If we are worried that this is a problem, I would rather create an issue in the Bootstrap project for this.

What do you think about this?

Reply:
Related issues in Bootstrap:

https://github.com/twbs/bootstrap/issues/15304
https://github.com/twbs/bootstrap/issues/22549#issuecomment-422023478

And maybe this: https://getbootstrap.com/docs/4.1/getting-started/accessibility/#color-contrast

Reply:
as per linked docs:

> Most colors that currently make up Bootstrap’s default palette—used throughout the framework for things such as button variations, alert variations, form validation indicators—lead to insufficient color contrast (below the recommended WCAG 2.0 color contrast ratio of 4.5:1) when used against a light background. <b>Authors will need to manually modify/extend these default colors to ensure adequate color contrast ratios.</b>

Considering highlighted recommendation, I think it's appropriate to modify when no theme is chosen

Reply:
Ok. Will do.